notification/src/handler/notification/notification.handler.js

```diff
@@ -181,13 +181,20 @@ function calculateUpdateActionsForPayment(payment, notification) {
       )
     else if (
       compareTransactionStates(oldTransaction.state, transactionState) > 0
-    )
+    ) {
       updateActions.push(
         getChangeTransactionStateUpdateAction(
           oldTransaction.id,
           transactionState
         )
       )
+      updateActions.push(
+        getChangeTransactionTimestampUpdateAction(
+          oldTransaction.id,
+          notificationRequestItem.eventDate
```

Comment:
Is the timezone of the notification always the same or depends on Adyen account? I wonder if we should use the same timezone or always convert to UTC as all data in commercetools?

Reply:
Commercetools only accepts UTC representation for transaction timestamp.
https://docs.commercetools.com/api/projects/payments#transaction
https://docs.commercetools.com/api/types#datetime

I don't think we can use same timezone as notification response.